src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -1,29 +1,60 @@
 ﻿namespace Microsoft.ApplicationInsights.AspNetCore.Extensions
 {
+    using System.Reflection;
+    using Microsoft.Extensions.Logging;
+
     /// <summary>
     /// Application Insights service options defines the custom behavior of the features to add, as opposed to the default selection of featuers obtained from Application Insights.
     /// </summary>
     public class ApplicationInsightsServiceOptions
     {
         /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationInsightsServiceOptions" /> class.
         /// Application Insights service options that controlls the default behavior of application insights features.
         /// </summary>
         public ApplicationInsightsServiceOptions()
         {
             this.EnableQuickPulseMetricStream = true;
             this.EnableAdaptiveSampling = true;
+            this.LoggerMinimumLevel = LogLevel.Information;
```

Comment:
When you resolve `ILogger` it goes over all loggerfactories registeged (AI logger factory would be one of them) and creates a logger saving it to array (https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging/Logger.cs#L27). `ILogger.Log` would go trough all the loggers and call message to them, so AI logger would get a message and write it to telemetry client.